lib/playground.dart

```diff
@@ -579,6 +582,22 @@ ${info['libraryName'] == null ? "" : "**Library:** ${apiLink == null ? info['lib
         messageSpan.classes.add('message');
         messageSpan.text = issue.message;
         e.children.add(messageSpan);
+        source.offset = issue.charStart;
+        dartServices.fix(source).timeout(serviceCallTimeout).then((FixesResponse fixResponse) {
+          if (fixResponse.fixes.isNotEmpty) {
```

Comment:
I don't think I understand the flow here; my understanding is that:

- the user hits ctrl-1
- we send out a request for fixes
- we display them as code completions
- if the user selects one, we perform the indicated fix

But I think you may have a different flow in mind?